ui/components/app/gas-details-item/index.scss

```diff
@@ -1,6 +1,6 @@
 .gas-details-item {
   &__gas-fee-warning {
-    color: var(--secondary-1);
+    color: var(--color-warning-default);
```

Comment:
Hmm unfortunately I think we may have to break some semantic rules here. 😅 
```suggestion
    color: var(--color-secondary-default);
```

Our warning color isn't really visible as text. Going to make a note and work towards fixing this. You used the token correctly. The token needs updating but that will take time.

<img width="704" alt="Screen Shot 2022-03-16 at 3 12 38 PM" src="https://user-images.githubusercontent.com/8112138/158701356-bab63050-f232-4f57-bc60-2458d432f0fd.png">